v1/kugo.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "Sony Xperia X Compact (F5321)",
+  "codename": "kugo",
+  "unlock": ["confirm_model", "upgrade_android"],
+  "user_actions": {
+    "recovery": {
+      "title": "Reboot to Recovery",
+      "description": "With the device powered off, hold Volume Down + Power.",
+      "image": "phone_power_down",
+      "button": true
+    },
+    "bootloader": {
+      "title": "Reboot to Bootloader",
+      "description": "With the device powered off, hold Volume Up and insert a USB cable. The LED indicator has to light up blue.",
+      "image": "phone_power_up",
+      "button": true
+    },
+    "boot": {
+      "title": "Boot the device",
+      "description": "Power on the device.",
+      "image": "phone_power_up",
+      "button": true
+    },
+    "confirm_model": {
+      "title": "Confirm your model",
+      "description": "Please double-check that your device is a Sony Xperia X Compact (F5321)."
+    },
+    "upgrade_android": {
+      "title": "Upgrade Android",
+      "description": "If the device is running Android, you might have to upgrade to Android 8.0.",
+      "link": "https://forums.ubports.com/topic/3229/sony-xperia-x-suzu-f5121"
+    }
+  },
+  "operating_systems": [
+    {
+      "name": "Ubuntu Touch",
+      "sanity_check": "Are you sure?",
+      "options": [
+        {
+          "var": "channel",
+          "name": "Channel",
+          "tooltip": "The release channel",
+          "link": "https://docs.ubports.com/en/latest/about/process/release-schedule.html",
+          "type": "select",
+          "remote_values": { "type": "systemimagechannels" }
+        },
+        {
+          "var": "wipe",
+          "name": "Wipe Userdata",
+          "tooltip": "Wipe personal data",
+          "type": "checkbox"
+        },
+        {
+          "var": "bootstrap",
+          "name": "Bootstrap",
+          "tooltip": "Flash system partitions using fastboot",
+          "type": "checkbox",
+          "value": true
+        }
+      ],
+      "prerequisites": [],
+      "steps": [
+        {
+          "type": "download",
+          "condition": {"var": "bootstrap", "value": true},
+          "group": "firmware",
+          "files": [
+            {
+              "url": "https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-kugo/lastSuccessfulBuild/artifact/halium-unlocked-recovery_kugo.img"
```

Comment:
It is not recommended to use the last successful build: What if you break recovery accidentially, and then users cannot use the installer anymore (Installer will boot this recovery for the installation process). Better, use e.g. https://ci.ubports.com/job/Device%20Compatibility%20Images/job/halium-kugo/129/artifact/halium-unlocked-recovery_kugo.img (Build 129) and I pin it for you, so it will never expire...

Reply:
Done, also rebased on ubports/master again.